### PR TITLE
created news page

### PR DESCRIPTION
**Pull request by Ksenaaa**

### CODE REVIEW

**Review by vell28 — APPROVED** [review]

File: routes/news.routes.js [hunk 1]
```diff
@@ -40,6 +40,23 @@ router.get(
   }
 )
 
+router.get(
+  '/news-ids',
+  async (req, res) => {
+    try {
+      let findNews = await newsModel.find()
```
Comment (vell28) [inline routes/news.routes.js]: can you try `newsModel.find().map`

File: routes/news.routes.js [hunk 2]
```diff
@@ -40,6 +40,23 @@ router.get(
...
+      
+      let manyNews = await findNews.map(item => ({
+        id: item._id
+      }))
+
+      res.json(manyNews)
```
Comment (vell28) [inline routes/news.routes.js]: manyNews = news or allNews

File: routes/news.routes.js [hunk 3]
```diff
@@ -8,13 +8,69 @@ router.get(
...
+  '/all-news',
+  async (req, res) => {
+    try {
+      let findNews = await newsModel.find()
+      
+      let allNews = await findNews.map(item => ({
+        id: item._id,
+        name: item.name,
+        img: item.pictures[0]
+      }))
```
Comment (vell28) [inline routes/news.routes.js:37]: why we need name and img? we will get it when we go to news details route

Comment (Ksenaaa) [reply routes/news.routes.js:37]: No, we need it in news page for news card

**Review by valentinPortside — APPROVED** [review]

**Pull request merged**